eng/pipelines/release-cli.yml

```diff
@@ -47,12 +47,19 @@ stages:
               SetShieldInfo: true
               GenerateCoverage: true
               AZURE_DEV_CI_OS: lin
+            LinuxARM64:
+              Pool: azsdk-pool-mms-ubuntu-2004-arm
+              # TODO: Is this a good name for the executable?
+              BuildTarget: azd-linux-arm64
```

Comment:
I think this name makes sense (and go uses the string `arm64` for `GOARCH` for this target, so it seems to match what the ecosystem expects)